pom.xml

```diff
@@ -85,7 +85,7 @@
     <sonar.version>6.7</sonar.version>
     <sonar-java.version>5.12.0.17701</sonar-java.version>
     <maven.checkstyle.plugin.version>2.17</maven.checkstyle.plugin.version>
-    <maven.sevntu.checkstyle.plugin.version>1.33.0</maven.sevntu.checkstyle.plugin.version>
+    <maven.sevntu-checkstyle-check.sevntu-checks.version>1.35.0</maven.sevntu-checkstyle-check.sevntu-checks.version>
```

Comment:
please revert property name. I am not comfortable changing them anymore in an unrelated issue.

Reply:
done